src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -72,7 +72,7 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
             boolean sessionIsOpen, int qnIdx, int responseIdx, String courseId, int totalNumRecipients) {
         return Templates.populateTemplate(
                 FormTemplates.TEXT_SUBMISSION_FORM,
-                Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                Slots.IS_SESSION_CLOSED, sessionIsOpen ? "false" : "true",
```

Comment:
This might be even more readable:
```java
Slots.IS_SESSION_CLOSED, Boolean.toString(!sessionIsOpen)
```
However, I actually prefer if we stick to `IS_SESSION_OPEN`, which is even more readable:
```java
Slots.IS_SESSION_OPEN, Boolean.toString(sessionIsOpen)
```
The final negation can happen at the point of usage (when setting the readonly field in JavaScript). When data is still in the process of being passed around, I feel it should be made as readable as possible.